### DISCUSSION TITLE
In what form does LPIPS and Inception Score expect data?

### DISCUSSION THREAD

**Discussion by Mahmood-Hussain** [Lightning-AI/torchmetrics#1226] [CompVision]
Hello, my question is I was looking at LPIPS and IS from the docs it is very confusing whether both the metrics expect the same form of data i.e normalized between [-1, 1] resized to [299, 299], unit8 

**INCEPTION SCORE DOC**

> Using the default feature extraction (Inception v3 using the original weights from [2]), the input is expected to be mini-batches of 3-channel RGB images of shape (3 x H x W) with dtype uint8. All images will be resized to 299 x 299 which is the size of the original training data.

**LPIPS DOC**

> Both input image patches are expected to have shape [N, 3, H, W] and be normalized to the [-1,1] range. The minimum size of H, W depends on the chosen backbone (see net_type arg).

I am using following code for transforms
```
transform = transforms.Compose([
    transforms.Lambda(lambda img: __resize(img, size, size)),
    transforms.ToTensor(),
    transforms.Normalize((0.5, 0.5, 0.5),(0.5, 0.5, 0.5)),
])
```
and then for Inception Score, I convert to unit8 like this 
`is_images.to(torch.uint8)`

I am doing it right way or I miss something?
